### DISCUSSION TITLE
Using Vico for Android 33 and older gradle wrapper 7.5

### DISCUSSION THREAD

**Discussion by harshjain-dev** [patrykandpatrick/vico#438] [Questions]
Hi, 
Currently are app is not fully move to Android 34, and haven't also updated cradle-wrapper to the latest. Is it possible to get a version which could work in lower gradle wrappers and supporting till Android 33

**Comment by patrickmichalik** [comment]
Hello! Please correct me if I’m wrong, but I don’t believe a Gradle version mismatch is problematic. Regarding the SDK version, the latest Vico version compiled against version 33 of the Android SDK is Vico 1.9.2. It’s worth noting, however, that `compileSdk` and `targetSdk` are independent. If your build fails when you add a Vico version compiled against version 34 of the SDK, it should be sufficient to set `compileSdk` to 34. `targetSdk` can remain unchanged. If you opt to use Vico 1.9.2, please keep in mind that some of the information in the wiki and API reference won’t apply, and we will, unfortunately, be unable to accept bug reports from you.
